basicOptions.lua

```diff
@@ -318,6 +318,10 @@ fctfloatmodeDropdown.initialize = function(dropdown)
 		info.func = function(self)
 			addon:SetCVar("floatingCombatTextFloatMode", self.value)
 			UIDropDownMenu_SetSelectedValue(dropdown, self.value)
+			if not InCombatLockdown() then
```

Comment:
Is InCombatLockdown() needed here? We already close the options when combat starts and they can't be opened in combat anyway.